packages/pretty-format/src/__tests__/AsymmetricMatcher-test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* eslint-disable max-len */
+
+'use strict';
+
+const prettyFormat = require('../');
+const AsymmetricMatcher = require('../plugins/AsymmetricMatcher');
+const jestExpect = require('jest-matchers');
```

Comment:
not really sure why this works. You need to make `jest-matchers` a devDependency of pretty-format.